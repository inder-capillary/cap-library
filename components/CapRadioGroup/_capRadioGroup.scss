@import "../styles/_variables.scss";


$radioButton:cap-radio-group-v2;
.ant-radio-group.#{$radioButton}{
    .ant-radio-checked .ant-radio-inner {
        background-color:map-get($CAP_PRIMARY, base);
    }
    
    .ant-radio-inner:after {
        background-color:$CAP_WHITE;
    
    }
}

.cap-radio-group{
  .cap-radio:not(:last-child){
    margin-right: 24px;
  }
}

.ant-radio-checked .ant-radio-inner {
    background-color:map-get($CAP_PRIMARY, base);
}

.ant-radio-inner:after {
    background-color:$CAP_WHITE;

}