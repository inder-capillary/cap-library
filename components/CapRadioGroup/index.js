/**
* CapRadioGroup
*/
import React from "react";
import PropTypes from 'prop-types';
import { Radio } from "antd";
import "./_capRadioGroup.scss";
const RadioGroup = Radio.Group;
const classNames = require('classnames');
export default class CapRadioGroup extends React.Component { // eslint-disable-line react/prefer-stateless-function
  render() {
    const {className, children, ...rest} = this.props;
    return (
      <RadioGroup {...rest} className={classNames("cap-radio-group", className)}>
        {children}
      </RadioGroup>
    );
  }
}

CapRadioGroup.propTypes = {
  children: PropTypes.any,
  className: PropTypes.string,
};
