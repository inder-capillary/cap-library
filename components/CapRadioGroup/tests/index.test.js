import renderer from 'react-test-renderer';
import React from 'react'; // eslint-disable-line no-unused-vars

import CapRadioGroup from '../index';

describe('<CapRadioGroup />', () => {
  it('CapRadioGroup renders correctly', () => {
    const tree = renderer.create(<CapRadioGroup />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
