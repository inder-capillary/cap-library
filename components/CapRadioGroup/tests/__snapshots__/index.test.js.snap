// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CapRadioGroup /> CapRadioGroup renders correctly 1`] = `
<div
  className="ant-radio-group ant-radio-group-outline cap-radio-group"
/>
`;
